### DISCUSSION TITLE
Search console: Page Resources - couldn't be loaded

### DISCUSSION THREAD

**Discussion by prasadad** [vercel/next.js#21422] [Help]
Hi anyone who has a insight with this, need a little help guys, cannot figure out how to fix these errors on google search console.
 https://vigyaa.io/_next/static/IOv9APkKF2prR0XEd8iME/_buildManifest.js
![image](https://user-images.githubusercontent.com/4686144/105369287-6a7dab00-5c28-11eb-8a16-dabdc1f5e2e7.png)

Thanks

**Comment by meotimdihia** [comment]
Same here, it happens if I use self-hosted
But it works fine if I use Vercel.
I still can't find out the problems.

**Comment by siuvdlec** [comment]
Hi, I have the same issue. Maybe someone has found a solution or a workaround?

**Comment by NymphaisArt** [comment]
I have the same problem. Cannot find the solution. 
![image](https://github.com/user-attachments/assets/0adb43ad-3457-4d2d-9f1a-433f256344a4)

**Comment by pblochdev** [comment]
Anyone found solution ? I have the same issue on next15 app router.
